### PR TITLE
Add dotGo interviews drafts.

### PR DESCRIPTION
**Pull request by AlekSi**
What's left:
* change `draft = true` to `false`;
* replace `XXX` with the actual name;
* build it with old working version of Hugo.

mp3 files were uploaded.

### CODE REVIEW

File: content/episode/2018/02-05-115.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
++++
+date = "2018-02-05"
+draft = true
```
Comment (AlekSi) [inline content/episode/2018/02-05-115.md]: ✍️

File: content/episode/2018/02-05-115.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+size = 5118967
+length = "00:14:56"
+categories = ["интервью"]
++++
+
+Интервью с Jaana Burcu Dogan a.k.a. rakyll на dotGo 2017.
+
+В нём по какой-то причине Алексей называет Derek'а Parker'а Трэвисом. Приношу свои извинения.
+
+Звук в самом начале очень плох, но потом становится лучше. Благодарим XXX за помощь в восстановлении звука этого интервью.
```
Comment (AlekSi) [inline content/episode/2018/02-05-115.md]: ✍️

File: content/episode/2018/02-05-116.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
++++
+date = "2018-02-05"
+draft = true
```
Comment (AlekSi) [inline content/episode/2018/02-05-116.md]: ✍️

File: content/episode/2018/02-05-116.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+title = "Выпуск 116. dotGo 2017: интервью с sam'ом boyer'ом"
+link = "golangshow.com/cdn/episodes/116.mp3"
+size = 5725472
+length = "00:16:14"
+categories = ["интервью"]
++++
+
+Интервью с sam'ом boyer'ом на dotGo 2017.
+
+Звук от Алексея очень плох, но sam'а слышно хорошо. Благодарим XXX за помощь в восстановлении звука этого интервью.
```
Comment (AlekSi) [inline content/episode/2018/02-05-116.md]: ✍️

**Review by rumyantseva — DISMISSED** [review]
Ok!

**Pull request merged**